src/grg_sphinx_theme/theme/grg_sphinx_theme/theme.conf

```diff
@@ -5,4 +5,4 @@ pygments_style=
 sidebars=sidebar-nav-bs.html
 
 [options]
-variable=
+navbar_links=
```

Comment:
why it is empty?

Reply:
This needs to placed as `grg-sphinx-theme` should not be enforce any links. By putting this variable I enable a new option for the applications which will be using grg-sphinx-theme. It is not a default option provided by pydata.

The behavior is, if the options is empty it will default to pydata's automatic generation for the links according to toctree inclusion in the rst file.

Reply:
make sense. thanks !

PS: Does it work if you put `navbar_links={}`